### PR TITLE
[watermask] added 'wat' parameter

### PR DESCRIPTION
**Pull request by dyoussef**
This pull request allows to use a **raster** watermask.
The parameter 'wat' is added (same level as 'cld' and 'roi').
If 'wat' is None, the watermask is created from srtm tiles.

### CODE REVIEW

File: python/config.py [hunk 1]
```diff
@@ -29,7 +29,7 @@
 
 # max number of tiles processed in parallel. None means the number of cores of
 # the cpu.
-cfg['max_nb_threads'] = None
+cfg['max_nb_threads'] = 1
```
Comment (carlodef) [inline python/config.py:32]: Hi David, thank you for the pull request. Why is it necessary to set the default number of threads to 1?

Comment (dyoussef) [inline python/config.py:32]: It is not necessary. It is a mistake (sorry)

Comment (carlodef) [inline python/config.py:32]: Ok no problem, I'll change it back to `None` before merging.

File: python/process.py [hunk 2]
```diff
@@ -271,8 +271,7 @@ def finalize_tile(tile_info, height_maps, utm_zone=None):
 
 
 def rectify(out_dir, A_global, img1, rpc1, img2, rpc2, x=None, y=None,
-            w=None, h=None, prv1=None, cld_msk=None,
-            roi_msk=None):
```
Comment (carlodef) [inline python/process.py:275]: Thanks for cleaning up these unused arguments!

**Pull request merged**